lib/cli/writer.js

```diff
@@ -107,8 +110,20 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
         if (!options.unmountOnSuccess) {
           return Bluebird.resolve();
         }
-
-        return mountutils.unmountDiskAsync(drive.device);
+        return new Bluebird((resolve, reject) => {
+          // Because Mac OS doesn't like immediate unmounting / ejection
```

Comment:
Very minor nit, but can you add a space above the comment for consistency purposes?